### ISSUE TITLE
PKC not showing up as casting target on iOS

### ISSUE THREAD

**Issue by niclasku** [croneter/PlexKodiConnect#1446]
PKC does not show up as a casting target in the Plex iOS App while it is working fine in the web client. I had a look into the Kodi log and found a lot of these messages:
```
2021-04-14 16:14:12.576 T:37893   DEBUG <general>: PLEX.listener: Serving GET request...
2021-04-14 16:14:12.576 T:37893   DEBUG <general>: PLEX.listener: remote request_path: resources
2021-04-14 16:14:12.577 T:37893   DEBUG <general>: PLEX.listener: params received from remote: {'X-Plex-Device-Name': 'iPhone von Niclas'}
2021-04-14 16:14:12.625 T:37895   DEBUG <general>: PLEX.listener: Serving GET request...
2021-04-14 16:14:12.626 T:37895   DEBUG <general>: PLEX.listener: remote request_path: resources
2021-04-14 16:14:12.626 T:37895   DEBUG <general>: PLEX.listener: params received from remote: {'X-Plex-Device-Name': 'iPhone von Niclas'}
2021-04-14 16:14:12.677 T:37897   DEBUG <general>: PLEX.listener: Serving GET request...
2021-04-14 16:14:12.677 T:37897   DEBUG <general>: PLEX.listener: remote request_path: resources
2021-04-14 16:14:12.677 T:37897   DEBUG <general>: PLEX.listener: params received from remote: {'X-Plex-Device-Name': 'iPhone von Niclas'}
2021-04-14 16:14:12.728 T:37899   DEBUG <general>: PLEX.listener: Serving GET request...
2021-04-14 16:14:12.729 T:37899   DEBUG <general>: PLEX.listener: remote request_path: resources
2021-04-14 16:14:12.729 T:37899   DEBUG <general>: PLEX.listener: params received from remote: {'X-Plex-Device-Name': 'iPhone von Niclas'}
2021-04-14 16:14:12.779 T:37901   DEBUG <general>: PLEX.listener: Serving GET request...
2021-04-14 16:14:12.780 T:37901   DEBUG <general>: PLEX.listener: remote request_path: resources
2021-04-14 16:14:12.780 T:37901   DEBUG <general>: PLEX.listener: params received from remote: {'X-Plex-Device-Name': 'iPhone von Niclas'}
```
[kodi.log](https://github.com/croneter/PlexKodiConnect/files/6311522/kodi.log)

**Comment by niclasku** [comment]
Changing this:

https://github.com/croneter/PlexKodiConnect/blob/808136bff893f54378f77ef1a040c02fdb321fe4/resources/lib/plexbmchelper/listener.py#L96-L97

To this:
```python
self.wfile.write(body.encode('utf-8'))
#self.wfile.close()
```

Seems to work for me. Not sure if something else breaks with these changes.

**Comment by croneter** [comment]
Ouh thanks!!

Are you using Kodi 19 Matrix?

**Comment by niclasku** [comment]
Yes, I am using Kodi 19. Thanks for your work on this addon :)

**Comment by MrDemous** [comment]
> Changing this:
> 
> https://github.com/croneter/PlexKodiConnect/blob/808136bff893f54378f77ef1a040c02fdb321fe4/resources/lib/plexbmchelper/listener.py#L96-L97
> 
> To this:
> 
> ```python
> self.wfile.write(body.encode('utf-8'))
> #self.wfile.close()
> ```
> 
> Seems to work for me. Not sure if something else breaks with these changes.

This worked on Android for me as well.

**Comment by croneter** [comment]
Should be fixed in PKC **BETA** version 3.1.2. Let me know here if this is not the case!

**Comment by MrDemous** [comment]
> Should be fixed in PKC **BETA** version 3.1.2. Let me know here if this is not the case!

Working great for me. Thanks @croneter

**Comment by niclasku** [comment]
Works for me. Thank you!
